File: src/test/java/dev/resmali/highlighter/SmaliHighlightingColorsTest.java
package dev.resmali.highlighter;

import com.intellij.openapi.editor.colors.TextAttributesKey;
import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class SmaliHighlightingColorsTest {
    @Test
    public void textAttributeKeysAreNamespacedByPluginId() {
        for (TextAttributesKey key : SmaliHighlightingColors.getAllKeys()) {
            assertTrue(
                    "Text attribute key is not namespaced: " + key.getExternalName(),
                    key.getExternalName().startsWith("dev.resmali."));
        }
    }
}

File: src/test/java/dev/resmali/highlighter/SmaliHighlightingColorsTest.kt
package dev.resmali.highlighter

import org.junit.Assert
import org.junit.Test

class SmaliHighlightingColorsTest {
    @Test
    fun textAttributeKeysAreNamespacedByPluginId() {
        for (key in SmaliHighlightingColors.allKeys) {
            Assert.assertTrue(
                "Text attribute key is not namespaced: ${key.externalName}",
                key.externalName.startsWith("dev.resmali."),
            )
        }
    }
}
